packages/bech32m/package.json

```diff
@@ -4,16 +4,18 @@
   "license": "(MIT OR Apache-2.0)",
   "description": "Tools for manipulating Penumbra bech32m strings",
   "type": "module",
+  "engine": {
+    "node": ">=22"
+  },
   "scripts": {
-    "build": "tsup src/*.ts --dts --minify --format esm",
-    "clean": "rm -rfv dist",
+    "build": "tsup --clean src/*.ts --dts --minify --keep-names --format esm",
+    "clean": "rm -rfv dist package penumbra-zone-*.tgz",
     "lint": "eslint src",
+    "prepack": "$npm_execpath run build",
     "test": "vitest run"
   },
```

Comment:
this is fairly representative of the scripts in most packages now.

i added --keep-names which i prefer, i expect consumers will bundle if they want mangled names.